launch_test.go

```diff
@@ -33,6 +33,8 @@ const (
 	TestSHA    = "abc123"
 )
 
+var TestParentBuildIDFloat interface{} = float64(1111)
```

Comment:
If it's not going to be updated, use a `const` instead of `var`

Reply:
I can't seem to assign that var as a `const`. I'm getting:
```
./launch_test.go:38: const initializer []float64 literal is not a constant
```

Reply:
Yeah, I guess it's because slices aren't immutable

Reply:
It's fine as is 😺